src/core/src/Context.cpp

```diff
@@ -214,7 +215,7 @@ void Context::reloadStage() {
     for (const auto& prim : stage->Traverse()) {
         const auto& path = prim.GetPath();
         if (UsdUtil::isCesiumTileset(path)) {
-            AssetRegistry::getInstance().addTileset(path);
+            AssetRegistry::getInstance().addTileset(path, pxr::SdfPath("/CesiumGeoreference"));
```

Comment:
It would be nice not to have to hardcode `"/CesiumGeoreference"` in multiple places. Would something like this work?

```c++
const auto georeference = UsdUtil::getOrCreateCesiumGeoreference();
AssetRegistry::getInstance().addTileset(path, georeference.GetPath());
```

Reply:
I was trying to go for the most performant route possible here. Ultimately, using `getOrCreateCesiumGeoreference` is going to require getting the stage again and getting the `CesiumGeoreference` object before we get what we need, which in this case is just the path.

It's worth pointing out, all of this for reload stage has to change in order for us to support multiple georeference origins, so I was leaning towards what will help us now rather than what will help us in the future.

I do agree with you that this shouldn't be hardcoded in multiple places though, and I've put the value in a static constant in `UsdUtil`.

Reply:
The static constant is fine, though I wouldn't be too concerned about performance in `reloadStage` since it's called infrequently.